server/src/utils/url.ts

```diff
@@ -0,0 +1,104 @@
+/*!
+ * Copyright 2021 - Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CheckURLResponse } from "../routes/apis.interfaces";
+import config from "../config";
+import { parse } from "tldts";
```

Comment:
This is really just cosmetics: I tried to keep the import from libraries separated by the import from local files, so we can immediately identify what we defined locally. Let's just add a blank line:

```suggestion
import { parse } from "tldts";

import config from "../config";
import { CheckURLResponse } from "../routes/apis.interfaces";
```